packages/checkout-storefront/src/hooks/useQueryVarsWithLocale.ts

```diff
@@ -0,0 +1,20 @@
+import { LanguageCodeEnum } from "@/checkout-storefront/graphql";
+import { useLocale } from "@/checkout-storefront/hooks/useLocale";
+import { DEFAULT_LOCALE } from "@/checkout-storefront/lib/regions";
+import { snakeCase } from "lodash-es";
+import { useCallback, useMemo } from "react";
+
+export const useQueryVarsWithLocale = () => {
```

Comment:
What does this function have to do with query variables? It doesn't use them, doesn't refer to them either.
It takes `data` that can be anything, spreads it inside an object, adds a languageCode to it and returns. No relation to query variables whatsoever.

Reply:
What I meant by that is `query/mutation` vars but I see how that may not be clear now you mentioned is so gonna change it to just a parser function for locale as you mentioned in comments below 👍